qiskit_nature/circuit/library/ansatzes/evolved_operator_ansatz.py

```diff
@@ -44,6 +44,14 @@ def __init__(
             name: The name of the circuit.
             initial_state: A `QuantumCircuit` object to prepend to the circuit.
         """
+        warnings.warn(
+            "This EvolvedOperatorAnsatz is deprecated as of 0.2.0, "
+            "and will be removed no earlier than 3 months after the release. "
+            "You should use the qiskit.circuit.library.EvolvedOperatorAnsatz "
+            "as a direct replacement instead.",
+            DeprecationWarning,
+            stacklevel=2,
+        )
```

Comment:
You may want to look into using `qiskit_nature.deprecation.warn_deprecated_same_type_name` instead of this manual warning :slightly_smiling_face:

Reply:
Oh... Thanks. I didn't notice it.